src/user-guide/running-workflows/reflow.rst

```diff
@@ -1,86 +1,227 @@
-``cylc trigger --reflow``. The flow number increments by one, each time.
+See below for suggested :ref:`use cases<flow-trigger-use-cases>` of this
+multi-flow capability, and a concrete :ref:`example<new-flow-example>`.
 
 .. note::
+   Flows :ref:`merge<flow-merging>` where (and if) tasks collide in the ``n=0``
+   :term:`active window`. Downstream of a merge, tasks are considered to belong
+   to all constituent flows.
 
-   By default ``cylc trigger`` does not start a new flow from the triggered
-   task. Use the ``--reflow`` option to start a new flow. :term:`Incomplete
-   tasks <incomplete task>` are the exception to this rule
-   because they are considered to be an unfinished part of the existing flow.
 
-Reflow Use Cases
-----------------
+Flow Numbers
+------------
 
-Reflows are useful if you need to re-wind your :term:`workflow` to allow
-it to evolve a new path into the future, or to repeat-run some subgraph
-to regenerate its outputs after making changes.
+Flows are identified by numbers passed down from parent task to child task in
+the graph.
 
-For example, the following :term:`cycling workflow` runs a :term:`task`
-called ``model`` in every cycle, followed by a postprocessing task, two
-product generating tasks, and finally a task that publishes results for
-the cycle point:
+Flow number ``1`` is triggered automatically by ``cylc play`` at :term:`scheduler`
+start-up. The next flow started by :ref:`manual triggering<triggering-flows>`
+gets the number ``2``, then ``3``, and so on.
 
-.. code-block:: cylc
+Tasks can carry multiple flow numbers as a result of
+:ref:`flow merging<flow-merging>` and :ref:`manual triggering<triggering-flows>`.
 
-   [scheduling]
-       cycling mode = integer
-       initial cycle point = 1
-       [[graph]]
-           P1 = model[-P1] => model => post => prod1 & prod2 => publish
+.. note::
+   Flow numbers are not yet exposed in the UI, but they are logged with task
+   events in the :term:`scheduler log`.
 
-Let's say the workflow has run to cycle 8, but we have just noticed that
-a corrupted ancillary file resulted in bad products at cycle 5.
 
-To rectify this we could fix the corrupted file and trigger a new flow
-(a reflow) from ``5/post``::
+.. _triggering-flows:
 
-   cylc trigger --reflow <workflow-id>//5/post
+Triggering & Flows
+------------------
 
-The new flow will regenerate and republish cycle 5 products before naturally
-coming to a halt, because the triggered tasks do not lead on to the next cycle
-and the rest of the graph.
+By default, manual triggering (with ``cylc trigger`` or the UI) starts a new
+:term:`front of activity<flow front>` in current flows.
+But it can also start new flows and trigger flow-independent single tasks.
 
-Meanwhile, the original flow will carry on unaffected, from cycle 8.
+In the diagrams below, the grey nodes represent tasks in flow ``1``, and
+the blue ones stem from a single manual triggering event.
```

Comment:
The blue ones can also represent tasks in flow 1.
How about:
The grey nodes represent tasks which run as a result of the original flow.
The blue nodes represent tasks which run as a result of the trigger.